dags/pull_wys.py

```diff
@@ -117,29 +119,25 @@ def pull_wys(ds=None):
 
     @task_group()
     def data_checks():
-        data_check_params = {
-            "table": "wys.speed_counts_agg_5kph",
-            "lookback": '60 days',
-            "dt_col": 'datetime_bin',
-            "threshold": 0.7
-        }
         check_row_count = SQLCheckOperatorWithReturnValue(
             task_id="check_row_count",
             sql="select-row_count_lookback.sql",
             conn_id="wys_bot",
-            params=data_check_params | {"col_to_sum": 'volume'},
-            retries=2
+            params={
```

Comment:
Can we print the error msg to slack if fails as well?

Reply:
Check out this change:
https://github.com/CityofToronto/bdit_data-sources/blob/6ac6b1344ea62b9956eeac6f2ae3d3ab1cf534a1/dags/custom_operators.py#L50-L59
Plus sample messages: https://bditto.slack.com/archives/C04BWRWVB3L/p1704297198015299